roles/ucp/templates/ucp.j2

```diff
@@ -62,9 +70,9 @@ stop)
     docker ps -a | grep 'ucp-' | awk '{print $1}' | xargs docker rm -v
 
     # XXX: do we need to uninstall ucp too?
-    #/usr/bin/docker run --rm -t --name ucp \
-    #    -v /var/run/docker.sock:/var/run/docker.sock \
-    #    docker/ucp uninstall --id=`cat {{ ucp_remote_dir }}/{{ ucp_instance_id_file }}` \
+    /usr/bin/docker run --rm -t --name ucp \
```

Comment:
I don't think we should enable this yet (unless it is recommended by docker folks)...from my experiments last time this seems a bit too heavy wait for service cleanup as it goes around removing the images etc forcing a fresh download next time which we si something we should avoid to keep provisioning time low.